### PR TITLE
Moving tolerance from the Namespace testing

### PR DESCRIPTION
**Pull request by loumalouomega**
The previous commit didn't include that

### CODE REVIEW

File: kratos/tests/math_utils/test_math_utils.cpp [hunk 1]
```diff
@@ -28,26 +28,16 @@ namespace Kratos
...
-//         }
-//         
+       
         /** Checks if the area of the triangle is calculated correctly using Heron equation.
          * Checks if the area of the triangle is calculated correctly using Heron equation.
          */
         
         KRATOS_TEST_CASE_IN_SUITE(MathUtilsHeronTest, KratosCoreMathUtilsFastSuite) 
         {
+            constexpr double TOLERANCE = 1e-6;
```
Comment (RiccardoRossi) [inline kratos/tests/math_utils/test_math_utils.cpp]: why is this capital? it is not a kratos variable...

**Review by RiccardoRossi — APPROVED** [review]
just change 

TOLERANCE -> tolerance
EPSILON -> epsilon

(if i recall correctly the style)

File: kratos/tests/math_utils/test_math_utils.cpp (hunk 1 above)
Comment (loumalouomega) [reply kratos/tests/math_utils/test_math_utils.cpp]: It was in one example of @pooyan-dadvand  or @roigcarlo

**Comment by roigcarlo** [comment]
I tend to put constexpr with capitals as if they were defines when they are only numbers, but I can change that in the other files as well

**Pull request merged**